src/kms/KmsApi.ts

```diff
@@ -28,11 +28,35 @@ interface UnwrapKeyResponse {
 
 export type BatchUnwrapKeyResponse = BatchResponse<UnwrapKeyResponse>;
 
+interface DeriveKeyResponse {
+    derivedKeys: DerivedKeys;
+}
+
+export type DerivedKeys = Record<string, DerivedKey[]>;
+
+export interface DerivedKey {
+    derivedKey: Base64String;
+    tenantSecretId: string; // TODO: worth using uuid type? worth using at all?
```

Comment:
We probably don't need it at all, and once changes are in the `secretNumericId` for the assignment id can change to something more useful.

Reply:
Yeah, I talked to Colt about removing it from the new response structure he's working on, so I'll end up removing it here as well